PowerShell/README.md

```diff
@@ -116,14 +116,35 @@ command's positional parameters. Always use the parameter/switch name, e.g. `Joi
 'build.ps1'` not `Join-Path $PSScriptRoot 'build.ps1'`. This improves readability and improves forward compatibility if
 a command's positional parameters change.
 
+# Importing Dependencies
+
+Always explicitly import functions from other modules, except function's from PowerShell's built-in modules, e.g.

```

Comment:
Maybe say `functions/cmdlets`? Some modules export cmdlets and not functions, i.e. `-Functions` doesn't do anything.